Observer/Observer/ConsoleObserver/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Net.Mentoring.Patterns.ConsoleObserver
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            ConsoleReader conReader = new ConsoleReader();
+            EventObserver eventObserver = new EventObserver(conReader);
+            Console.WriteLine("Inpute quit to stop");
+            do
+            {
+                conReader.InputString = Console.ReadLine();
+            } while (conReader.Flag);
```

Comment:
Sorry but didnt get what this code does? i guess it's just waiting for user input and that's it?

Reply:
Write a class which listens to console and raise an event if 'quit' is written in console